modules/serverless-create-eventing-namespace.adoc

```diff
@@ -31,12 +32,13 @@ image::serverless-create-namespaces.png[Namespaces page]
 image::create-eventing-namespace.png[Creating the `knative-eventing` namespace]
 . Click *Create*.
 
-[id="serverless-create-eventing-project-oc_{context}"]
-== Creating the `knative-eventing` namespace using the CLI
+== Creating the namespace using the CLI
```

Comment:
Oh now I see why. I think that you're supposed to have two separate modules in cases like these, instead of one module with two sets of procedures. Since it was already there, no worries about it for now, just for future consideration.